src/test/java/LoginTests.java

```diff
@@ -1,111 +1,51 @@
-
-        Assert.assertTrue(avatarIcon.isDisplayed());
-//        Assert.assertEquals(avatarIcon.isDisplayed(), true);
-
-        driver.quit();
-    }
-
-    @Test
-    public static void loginValidEmailEmptyPasswordTest() {
-
-//      Added ChromeOptions argument below to fix websocket error
-        ChromeOptions options = new ChromeOptions();
-        options.addArguments("--remote-allow-origins=*");
-
-        WebDriver driver = new ChromeDriver(options);
-        driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
-
-        String url = "https://bbb.testpro.io/";
-        driver.get(url);
-
-        WebElement emailField = driver.findElement(By.cssSelector("input[type='email']"));
-        emailField.click();//not needed
-        emailField.clear();
-        emailField.sendKeys("demo@class.com");
-
-        WebElement passwordField = driver.findElement(By.cssSelector("input[type='password']"));
-        passwordField.click();//not needed
-        passwordField.clear();
-        passwordField.sendKeys("");
-
-        WebElement submit = driver.findElement(By.cssSelector("button[type='submit']"));
-        submit.click();
-
-        Assert.assertEquals(driver.getCurrentUrl(), url); //https://bbb.testpro.io/
-
-        driver.quit();
-    }
-}
+//import org.openqa.selenium.By;
+//import org.openqa.selenium.WebDriver;
+//import org.openqa.selenium.WebElement;
+//import org.openqa.selenium.chrome.ChromeDriver;
+//import org.openqa.selenium.chrome.ChromeOptions;
+//import org.testng.Assert;
+//import org.testng.annotations.AfterMethod;
+//import org.testng.annotations.BeforeMethod;
+//import org.testng.annotations.Test;
+//
+//import java.time.Duration;
+//
+//public class LoginTests extends BaseTest {
+//
+//
+//    @Test
+//    public void loginInvalidEmailValidPasswordTest(){
+//
+//        navigateToPage();
+//        provideEmail("invalid@class.com");
+//        providePassword("te$t$tudent");
+//        clickSubmit();
+//
+//        Assert.assertEquals(driver.getCurrentUrl(), url); //https://bbb.testpro.io/
+//
+//    }
+//    @Test
+//    public void loginValidEmailPasswordTest(){
+//        navigateToPage();
+//        provideEmail("demo@class.com");
+//        providePassword("te$t$tudent");
+//        clickSubmit();
+//        isAvatarDisplayed();
+//    }
+//    @Test
+//    public static void loginValidEmailEmptyPasswordTest() {
+//        navigateToPage();
+//        provideEmail("demo@class.com");
+//        providePassword("");
+//        clickSubmit();
+//
+//        Assert.assertEquals(driver.getCurrentUrl(), url); //https://bbb.testpro.io/
+//    }
+//
+//    public static void isAvatarDisplayed() {
+//        WebElement avatarIcon = driver.findElement(By.cssSelector("img[class='avatar']"));
+//        Assert.assertTrue(avatarIcon.isDisplayed());
+////        Assert.assertEquals(avatarIcon.isDisplayed(), true);
+//    }
+//
```

Comment:
**I recommend not using code comments:** for, example, when you have a code that cannot be understood, then comments are not the solution. Try to rewrite the code or rename fields and other elements in a way that the reader will understand the action that you are doing there.